packages/core/src/blocks/ImageBlockContent/ImageBlockContent.ts

```diff
@@ -130,7 +146,14 @@ export const imageToExternalHTML = (
   if (block.props.showPreview) {
     image = document.createElement("img");
     image.src = block.props.url;
-    image.alt = block.props.name || block.props.caption || "BlockNote image";
+    // Accessibility for exported HTML: prefer caption as alt when present
+    if (block.props.caption) {
+      image.alt = block.props.caption;
+    } else {
+      image.alt = "";
+      image.setAttribute("role", "presentation");
+      image.setAttribute("aria-hidden", "true");
+    }
```

Comment:
Missing `block.props.name`, why not just:
```suggestion
	image.alt = block.props.name || block.props.caption || "";
    if (!image.alt) {
      image.setAttribute("role", "presentation");
      image.setAttribute("aria-hidden", "true");
    }
```